README.md

```diff
@@ -208,28 +207,25 @@ In order to update your repository:
    1. Update the git repository. (With `git pull` or anything else)
    2. Make sure that Randovania is closed.
    3. Re-run the steps from "Getting Started", starting at step 2.
-      1. In case of unexpected errors, delete the `venv` in the root of the repository and start again.
+      1. In case of unexpected errors, delete the `.venv` in the root of the repository and start again.
```

Comment:
does windows hide dotfiles by default? I know most linux file managers do. so we may want to add an aditional sentence of something like "enable the setting to show hidden files in your file manager"

Reply:
I don't even like this mention here, and hopefully uv can actually handle things better. I also don't like the venv being .venv, maybe I can configure it.

Reply:
i know that .venv is *technically* the standard nowadays but i do prefer venv for visibility purposes.
Especially with how often our peeps have issues with some weird dependency where the easiest thing to do is just delete the venv and reinstall deps....